repos/spack_repo/builtin/packages/py_graphene_tornado/package.py

```diff
@@ -1,31 +0,0 @@
-# Copyright Spack Project Developers. See COPYRIGHT file for details.
-#
-# SPDX-License-Identifier: (Apache-2.0 OR MIT)
-
-from spack_repo.builtin.build_systems.python import PythonPackage
-
-from spack.package import *
-
-
-class PyGrapheneTornado(PythonPackage):
```

Comment:
Considering that more recent versions of py-graphene-tornado have removed all version constraints on werkzeug, I think it's best if we do the same, and we can deprecate this package/version later if it proves not to work with any now buildable versions of werkzeug (and hopefully add new versions, but if it turns out no one cares, we can delete it).

Reply:
Volunteers are welcome to add new versions of the package.

Reply:
I don't think this requires adding any new version -- I think the version constraint is not required, based on what I see from newer versions. I'll test that out.